### ISSUE TITLE
Will this be available as a vscode extension one day?

### ISSUE THREAD

**Issue by mishka** [zd-zero/waifu-motivator-plugin#207]
I really like this extension and I was curious if you'd be interested in supporting vscode in the future

**Comment by Unthrottled** [comment]
:thinking: 

I've thought about this before, the `Waifu of the Day` feature would be easy to replicate. That would be just adding a [WebView](https://code.visualstudio.com/api/extension-guides/webview)

However the annoying part is that VSCode has some annoying [restrictions](https://code.visualstudio.com/api/extension-capabilities/overview#no-dom-access) which may prevent the motivation events from being implemented. Either that, or it may need to be re-imagined. 

What do you thing @zd-zero ?

**Comment by zd-zero** [comment]
Definitely, as we've already aimed to decouple the assets from here, we can have some other platform consumers. I just can't say anything about the VSCode as I haven't seen the SDK or even the event system from it. But yeah, we'll just figure it out as our cause is to have more Waifu motivate us in different places! 😂

Given that, we may need to have a centralized system for the clients to consume it (Waifu as a Service?), even for web extension 😅. I also think that the IJ plugin is getting stable and reaching its purpose (enough to motivate us developers) but what I see is its lacking content, so we should start binge watch more anime 🤪. I may find more assets 🕵️‍♂️

**Comment by CallyyllaC** [comment]
Would love to have this for VS as well

**Comment by SanariSan** [comment]
+1 for vscode
[Doki theme](https://github.com/doki-theme/doki-theme-vscode) with static sticker is not enough!
